app/assets/stylesheets/_layout.scss

```diff
@@ -0,0 +1,19 @@
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+figure {
+  margin: 0;
+}
+
+img,
+picture {
```

Comment:
Interesting. I think I’m not quite seeing how `box-sizing` is something related to media…it applies to everything in the layout of the website/app.